NSS-Day-Cohort-42/rare-media-moguls
# Users can now logout.

#### Changes Made
1. Renamed file `NavBar.js` to `Nav.js` and updated filename in related files.
1. Modified `Nav.js` to include a logout button.

#### Steps to Review
1. Checkout this branch locally and run the application.
```
git fetch --all
git checkout mp-logout
```
2. Run the application.
```
npm start
```
3. Open a new Terminal tab (⌘T) and navigate to the server directory.
```
..
server
```
4. Initialize virtual environment, and run the server.
```
pipenv shell
watchgod request_handler.main
```
5. Test app functionality. 
> Open dev tools in the application and select `Application` from the menu at the top.
> Open Local Storage in the left sidebar of dev tools, and select the URL shown.
> Enter `rare_user_id` as the Key and `1` as the Value, then refresh the page.
> You should be redirected to the home page as a logged in user.
> Click the Logout button.
> You should be redirected to the login page and the local storage should be cleared.
6. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
